scripts/simplify.py

```diff
@@ -0,0 +1,215 @@
+#!/usr/bin/python3
+
+import sys
+import getopt
+import uuid
+from xml.dom import minidom
+
+default_layer_attrs = {
+	'inkscape:groupmode': 'layer',
+	'sodipodi': 'true', # Force layer locked by default
+	'style': 'display:inline'
+}
+
+default_trace_style = 'fill:none;stroke:#ff0000;stroke-width:1.25;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1'
```

Comment:
Should move the default attributes into dicts and generate string on the fly.

Reply:
80d27b76b7b43083975c533d0158ded271fc5abb